projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -349,6 +349,8 @@ private function build_task( $task ) {
 		$built_task['completed']    = $this->is_task_complete( $task );
 		$built_task['disabled']     = $this->is_task_disabled( $task );
 		$built_task['subtitle']     = $this->load_subtitle( $task );
+		$built_task['content']      = $this->load_value_from_callback( $task, 'content_callback' );
+		$built_task['actions']      = $this->load_value_from_callback( $task, 'actions_callback' );
```

Comment:
I have a concern around how we're modelling this that will cause complications down the line. As things stand we have a list of contexts and a single set of content and actions across all those contexts. From a longer-term perspective, would we ever want specific content or actions to be linked to particular contexts? Or will we always want the same content and actions across all contexts? I don't know the answer to this question, but we might want to set this up so the content is specifically linked to a context, which means we might want the JSON representation to look something more like the following:
```js
{
	/* standard task properties */
	"extended_content": {
		"context_example": {
			"content": { /* content */ },
			"actions": [ /* actions */ ],
			/* Maybe we need an additionalContexts array here? */
		},
		/* Other contexts with separate content/actions would have dedicated sections  */
	}
}
```